node/node.go

```diff
@@ -408,28 +406,30 @@ func (n *Node) printTxSentCounter(ctx context.Context) {
 	totalTxCounter := uint64(0)
 	counterSeconds := 0
 
-	select {
-	case <-time.After(time.Second):
-		txSent := atomic.SwapUint32(&n.txSentCounter, 0)
-		if txSent > maxTxCounter {
-			maxTxCounter = txSent
-		}
-		totalTxCounter += uint64(txSent)
+	for {
```

Comment:
so before this, the print was displayed only once?

Reply:
yes, the refactor in this area was not correct.